pkgs/top-level/aliases.nix

```diff
@@ -1021,6 +1021,7 @@ mapAliases ({
   pdf2htmlEx = throw "pdf2htmlEx has been removed from nixpkgs, as it was unmaintained"; # Added 2020-11-03
   pdfmod = throw "pdfmod has been removed"; # Added 2022-01-15
   pdfread = throw "pdfread has been remove because it is unmaintained for years and the sources are no longer available"; # Added 2021-07-22
+  pentablet-driver = throw "pentablet-driver has been renamed to xp-pen-g430-driver"; # Added 2022-06-23
```

Comment:
I think it's preferred to make an alias for a while first, and then converting it to a throw later.

Reply:
Wait, so should I make this and `xp-pen-g430` both throw, or this and `xp-pen-g430` both be aliases?

Reply:
> Wait, so should I make this and xp-pen-g430 both throw, or this and xp-pen-g430 both be aliases?

`pentablet-driver` should be an alias, while `xp-pen-g430` should be a throw. 

> (I personally believe it should be preferred to make it a throw, since it's a simple search-and-replace in the configuration, and there doesn't appear to be any 'warn' option)

Aliases actually throw a warning when a user attempts to rebuild their system with one in the environment, IIRC. Because of that we prefer to have it be an alias for a while so that the users can migrate to the new package name without breaking existing configurations.